crates/epaint/src/shape.rs

```diff
@@ -685,6 +695,10 @@ pub struct TextShape {
     /// This will NOT replace background color nor strikethrough/underline color.
     pub override_text_color: Option<Color32>,
 
+    /// If set, the text color will be gamma multiplied by the given factor
+    /// This will NOT replace background color nor strikethrough/underline color.
+    pub gamma_multiply: Option<f32>,
```

Comment:
Is there any difference between `None` and `Some(1.0)`? If not, I suggest just using a `f32` here defaulting to `1.0`, and testing against in the tessallalator for early-out-optimization